### PR TITLE
LPS-82491 Intellij support in target platform

### PR DESCRIPTION
**Pull request by gamerson**
Can you please publish these please? 
1. `gradle-plugins-target-plataform`
2. `gradle-plugins-workspace` 

Thanks!

### CODE REVIEW

**Comment by gamerson** [comment]
ci:test:relevant

File: modules/sdk/gradle-plugins-target-platform/CHANGELOG.markdown [hunk 1]
```diff
@@ -13,14 +13,16 @@
 	- configurations added by [Liferay Gradle Plugins Test Integration]
 	- configurations added by the `java` plugin
 
-## 1.1.0 - *(Unreleased)*
+## 1.1.0 - 2018-07-03
 
 ### Added
 - [LPS-81530]: Add the ability to configure imported BOMs to manage Java
 dependencies and the various artifacts used in resolving OSGi dependencies.
+- [LPS-82491]: Add support for IntelliJ Idea plugin
```
Comment (Ithildir) [inline modules/sdk/gradle-plugins-target-platform/CHANGELOG.markdown]: I think we have the convention to write something like this instead:
> the `idea` plugin.

Don't forget the period at the end.

File: modules/sdk/gradle-plugins-target-platform/src/main/java/com/liferay/gradle/plugins/target/platform/TargetPlatformIDEPlugin.java [hunk 2]
```diff
@@ -54,7 +63,8 @@
 	@Override
 	public void apply(Project project) {
 		GradleUtil.applyPlugin(project, EclipsePlugin.class);
-		GradleUtil.applyPlugin(project, JavaBasePlugin.class);
+		GradleUtil.applyPlugin(project, JavaPlugin.class);
+		GradleUtil.applyPlugin(project, IdeaPlugin.class);
```
Comment (Ithildir) [inline modules/sdk/gradle-plugins-target-platform/src/main/java/com/liferay/gradle/plugins/target/platform/TargetPlatformIDEPlugin.java]: Sort these, please.

File: modules/sdk/gradle-plugins-target-platform/CHANGELOG.markdown [hunk 3]
```diff
@@ -13,14 +13,16 @@
...
 
 [Liferay Gradle Plugins]: https://github.com/liferay/liferay-portal/tree/master/modules/sdk/gradle-plugins
 [Liferay Gradle Plugins Test Integration]: https://github.com/liferay/liferay-portal/tree/master/modules/sdk/gradle-plugins-test-integration
 [LPS-77343]: https://issues.liferay.com/browse/LPS-77343
 [LPS-80222]: https://issues.liferay.com/browse/LPS-80222
-[LPS-81530]: https://issues.liferay.com/browse/LPS-81530
\ No newline at end of file
+[LPS-81530]: https://issues.liferay.com/browse/LPS-81530
```
Comment (Ithildir) [inline modules/sdk/gradle-plugins-target-platform/CHANGELOG.markdown]: Please update the `README.markdown` as well.

**Comment by gamerson** [comment]
Hey @Ithildir  I'm back from PTO, I've made those changes.

**Comment by gamerson** [comment]
resent here: https://github.com/Ithildir/liferay-portal/pull/1065

**Pull request closed without merging**